releases.yaml

```diff
@@ -0,0 +1,16 @@
+sources:
+  - id: keycloak
+    productName: Keycloak
+    repo: keycloak/keycloak
+    githubReleaseFrom: 23.0.0
+    releaseNotes: release/${version.getVersionShorter()}/docs/documentation/release_notes/topics/${version.getVersion()?replace(".", "_")}.adoc
+    attributeSources:
+      - release/${version.getVersionShorter()}/docs/documentation/topics/templates/document-attributes.adoc
+      - ${version.getVersion()}/docs/documentation/topics/templates/document-attributes.adoc
+    migrationGuidePath: docs/latest/upgrading/index.html#migration-changes
+  - id: keycloak-client
+    productName: Keycloak Client Libraries
+    repo: keycloak/keycloak-client
+    githubReleaseFrom: 26.0.1
```

Comment:
Would remove `githubReleaseFrom` for clients, and make it optional instead. Since for clients it's all releases ;)

Reply:
Done